main.nf

```diff
@@ -53,7 +54,7 @@ def helpMessage() {
       --pico                        Sets trimming and standedness settings for the SMARTer Stranded Total RNA-Seq Kit - Pico Input kit. Equivalent to: --forward_stranded --clip_r1 3 --three_prime_clip_r2 3
       --fcExtraAttributes           Define which extra parameters should also be included in featureCounts (default: gene_names)
       --fcGroupFeatures             Define the attribute type used to group features. (default: 'gene_name')
-      --fcGroupFeaturesType         Define the type attribute used to group features based on the group attribute (default: 'gene_biotype')
+      --fcGroupFeaturesType         Define the type attribute used to group features based on the group attribute (default: 'gene_biotype', compatible with iGenomes and ENSEMBL. Change to 'gene_type' for GENCODE)
```

Comment:
Nice 👍 That was already something that came up with some of our project managers, so this should help :-)